README.md

```diff
@@ -24,6 +24,18 @@ optional arguments:
   -v VERSION, --version VERSION
                         Version (default=latest)
 ```
+Example:
+```bash
+# cd artifacthub
+python3 generate_config.py -r keptn-sandbox/datadog-service -v 0.2.0
+```
+Use pip to install the dependencies like this:
+```bash
+# cd artifacthub
+pip install -r requirements.txt 
+```
```

Comment:
I would move this part above the example, as it is more or less a requirement for the script to work without problems. A leading paragraph like: `Before the script can be executed the first time, the required dependencies have to be installed via pip: ` would also be nice

Reply:
Fully agree. Though I would also advise to work with Python virtual environments or alike, rather than installing dependencies globally (which, under certain older systems could even require sudo)

Reply:
Do you think we should add a line around this?